docs/Cmdlets/Get-ScriptAnalyzerRule.md

```diff
@@ -1,7 +1,7 @@
 ---
 external help file: Microsoft.Windows.PowerShell.ScriptAnalyzer.dll-Help.xml
 Module Name: PSScriptAnalyzer
-ms.custom: PSSA v1.20.0
+ms.custom: PSSA v1.21.0
```

Comment:
Is this ok/correct in preparation for the next release @sdwheeler ?

Reply:
@bergmeister Yes, this is fine. When we release it, the docs need to be copied from the source repo to the docs repo.